go.mod

```diff
@@ -11,62 +11,71 @@ require (
 	github.com/aws/aws-sdk-go v1.44.93
 	github.com/go-logr/logr v1.2.0
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.23.0
-	k8s.io/apimachinery v0.23.0
-	k8s.io/client-go v0.23.0
+	k8s.io/api v0.24.0
+	k8s.io/apimachinery v0.24.0
+	k8s.io/client-go v0.24.0
```

Comment:
Please revert the changes made in `go.mod` and `go.sum` files.

Reply:
Done